staging/cse/windows/README

```diff
@@ -13,8 +13,8 @@
 1. Run below commands to build a test package
 ```bash
 branchName="master"
-currentCseVersion="v0.0.27" # `WindowsCSEScriptsPackage` defined in `parts/windows/kuberneteswindowssetup.ps1`
-testCseVersion="v0.0.27.1" # Test package name. NOTE: Please do not use the official package format and earlier used version.
+currentCseVersion="v0.0.28" # `WindowsCSEScriptsPackage` defined in `parts/windows/kuberneteswindowssetup.ps1`
```

Comment:
should be 29

Reply:
Updated. It is used for test purpose but I added it in the commit. It should be better to update it every time to bump cse script package version.